### DISCUSSION TITLE
BurntToast v1 Preview2 Feedback

### DISCUSSION THREAD

**Discussion by Windos** [Windos/BurntToast#218] [General]
Please use this thread for any and all feedback regarding BurntToast v1 Preview2, including:

* Questions
* Comments
* Requests for changes in Preview3
* Prioritizations of features to be added in upcoming previews
* Anything else related to the current state of v1

**Comment by Maneridiet** [comment]
I greatly appreciate your hard work! I'm wondering if there's a way to utilize the BurntToast Preview function in an "alarm" context. My goal is to establish a notification system that reaches every user, irrespective of their current focus assist settings.

**Comment by wi-fr** [comment]
I would love trying out an Add-BTProgressBar function in the next preview of v.1 that approximates the functionality provided by v.0.8.5

```
$ProgressBar1 = New-BTProgressBar -Status 'Starting...' -Value 0
$ProgressBar1.Status.BindingName = 'ProgressInfo'
$ProgressBar1.Title = 'HorizontalBarText'

$Binding1 = New-BTBinding -Children  $ProgressBar1
$Visual1 = New-BTVisual -BindingGeneric $Binding1
$Content1 = New-BTContent -Visual $Visual1 -Duration Long

Submit-BTNotification -Content $Content1 -UniqueIdentifier 'MyApp' -DataBinding @{
  ProgressInfo      = ''
  HorizontalBarText = ''
}

$n = 25
for ($i = 1; $i -le $n; $i++) {
  $null = Update-BTNotification -UniqueIdentifier 'MyApp'-DataBinding @{
    0                 = $i/$n
    ProgressInfo      = "Processing page $i"
    HorizontalBarText = "<$i>"
  }
  Start-Sleep 0.2
}
```

**Comment by God-damnit-all** [comment]
> Anything else related to the current state of v1

@Windos I've been quietly watching the project for a while now and I realized I really needed a notification popup for something I'm working on today. It seems you (somewhat) recently removed the bit about v1 from the README. What's going on? Has the project been abandoned?

**Reply by Windos** [reply]
Hey @God-damnit-all, things are still progressing... just **much** slower than I'd like.

That v1 notice I removed is because I ended up deciding against doing a full re-write of the module so v1 isn't going to be as large of a breaking change as it was originally looking like. This meant having a big banner warning people that if they upgrade to v1 it'll break all their existing scripts wasn't warranted any more.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
